src/arrayMethodSort.js

```diff
@@ -1,11 +1,42 @@
+/* eslint-disable prettier/prettier */
 'use strict';
 
 /**
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
Using `[].__proto__` to extend the array prototype is not recommended. It's better to use `Array.prototype` directly, like `Array.prototype.sort2 = function(compareFunction) { ... }` for clarity and to avoid potential issues.